<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-03-21T10:29:20Z" name="Suite" finished-at="2024-03-21T10:29:46Z" duration-ms="26173">
    <groups>
    </groups>
    <test started-at="2024-03-21T10:29:20Z" name="Test" finished-at="2024-03-21T10:29:46Z" duration-ms="26173">
      <class name="com.mystore.testcases.TC_MyAccountPageDDT">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.mystore.testcases.TC_MyAccountPageDDT@1583741e]" started-at="2024-03-21T15:59:20Z" name="setup" finished-at="2024-03-21T15:59:35Z" duration-ms="15232" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="VerifyLogin(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.mystore.testcases.TC_MyAccountPageDDT@1583741e]" started-at="2024-03-21T15:59:38Z" name="VerifyLogin" data-provider="LoginDataProvider" finished-at="2024-03-21T15:59:43Z" duration-ms="5034" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[jd2@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[jd@123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[John Doe]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyLogin -->
        <test-method signature="VerifyLogin(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.mystore.testcases.TC_MyAccountPageDDT@1583741e]" started-at="2024-03-21T15:59:43Z" name="VerifyLogin" data-provider="LoginDataProvider" finished-at="2024-03-21T15:59:45Z" duration-ms="2572" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[jd2@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[jd@123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[John Lewis]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.mystore.testcases.TC_MyAccountPageDDT.VerifyLogin(TC_MyAccountPageDDT.java:58)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyLogin -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.mystore.testcases.TC_MyAccountPageDDT@1583741e]" started-at="2024-03-21T15:59:45Z" name="tearDown" finished-at="2024-03-21T15:59:46Z" duration-ms="545" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.mystore.testcases.TC_MyAccountPageDDT -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
